src/App/Pages/Accounts/TwoFactorPage.xaml.cs

```diff
@@ -29,6 +31,7 @@ public TwoFactorPage(bool? authingWithSso = false, AppOptions appOptions = null,
             _orgIdentifier = orgIdentifier;
             _broadcasterService = ServiceContainer.Resolve<IBroadcasterService>("broadcasterService");
             _messagingService = ServiceContainer.Resolve<IMessagingService>("messagingService");
+            _vaultTimeoutService = ServiceContainer.Resolve<IVaultTimeoutService>("vaultTimeoutService");
```

Comment:
⛏️ No need for the hardcoded string key
```suggestion
            _vaultTimeoutService = ServiceContainer.Resolve<IVaultTimeoutService>();
```